README.md

```diff
@@ -1,5 +1,15 @@
 # cargo-ament-build
-
+## getting started.
+At the moment, this tool only works with existing freight packages. Here is an example you can try yourself:
```

Comment:
I'm not sure what you mean by `freight` here, could you elaborate? Thanks.

Reply:
I meant `cargo`

Reply:
I think my modified version of [cargo-ament-build](https://github.com/tier4/cargo-ament-build) should not be merged now.

My [cargo-ament-build](https://github.com/tier4/cargo-ament-build) is for [safe_drive](https://github.com/tier4/safe_drive), which is a formally verified ROS 2 client library in Rust. See, [my paper](https://github.com/tier4/safe_drive/blob/main/publications/Deadlock-_and_Starvation-free_Formally_Verified_Client_Library_for_Robots_IEEE_ICSRC_2023.pdf) for detai.

My [cargo-ament-build](https://github.com/tier4/cargo-ament-build) parses IDL files and generate message types in Rust for [safe_drive](https://github.com/tier4/safe_drive), and it is performed by using [safe_drive_msg](https://github.com/tier4/safe_drive_msg) and [t4_idl_parser](https://github.com/tier4/idl_parser).

If the official ros2-rust/cargo-ament-build will also support [safe_drive](https://github.com/tier4/safe_drive), I will fix to support both ros2-rust and safe_drive, and send a PR.

My [cargo-ament-build](https://github.com/tier4/cargo-ament-build) supports Windows and [Cargo's workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html).
So, I think there are some merits for the official ros2-rust/cargo-ament-build.

Reply:
I don't know what you're trying to tell me. I just added some stuff to the readme.

Reply:
@ytakano thanks for chiming in. However, I don't fully understand the rationale for not at least trying to submit the features of your fork here. By working on it by yourself you'd be missing on all the improvements that the rest of the `cargo-ament-build` team is working on.

`cargo-ament-build` can be used without `ros2-rust`, so you're free to continue your work on `safe_drive`, it's unfortunate to duplicate efforts on `cargo-ament-build`

Reply:
So I'm completely overwhelmed by the situation. I have no idea how to separate my pull request from other commits. My commits simply relate to a completely different file. I also thought that this would not be a problem because I simply forked the main branch of `ros2_rust`.

Reply:
@esteve Thank you.

@Guelakais 
I have tried to fork and it does not contain my updates.
https://github.com/ytakano/cargo-ament-build

Reply:
So do you mean I should just trie to fork it again?

Reply:
> it's unfortunate to duplicate efforts on cargo-ament-build

@esteve Handling IDL files was a problem when implementing safe_drive. To solve it, I modified [cargo-ament-build](https://github.com/ros2-rust/cargo-ament-build) for specializing safe_drive.

As a result of this, my version has became less portable than the original.
So, I have not sent a PR.

Reply:
> So do you mean I should just trie to fork it again?

Yes. I think.